### ISSUE TITLE
TrackPlayer.getPosition doesn't return updated value after seekTo

### ISSUE THREAD

**Issue by nishanBende** [DoubleSymmetry/react-native-track-player#674]
# Configuration
 React Native Environment Info:
    System:
      OS: macOS 10.14.5
      CPU: (4) x64 Intel(R) Core(TM) i5-4278U CPU @ 2.60GHz
      Memory: 20.15 MB / 8.00 GB
      Shell: 5.3 - /bin/zsh
    Binaries:
      Node: 10.16.0 - ~/.nvm/versions/node/v10.16.0/bin/node
      Yarn: 1.16.0 - ~/.nvm/versions/node/v10.16.0/bin/yarn
      npm: 6.9.0 - ~/.nvm/versions/node/v10.16.0/bin/npm
      Watchman: 4.9.0 - /usr/local/bin/watchman
    SDKs:
      iOS SDK:
        Platforms: iOS 12.2, macOS 10.14, tvOS 12.2, watchOS 5.2
    IDEs:
      Xcode: 10.2.1/10E1001 - /usr/bin/xcodebuild
    npmPackages:
      react: 16.8.3 => 16.8.3 
      react-native: 0.59.4 => 0.59.4 
    npmGlobalPackages:
      react-native-cli: 2.0.1
      react-native: 0.59.4

react-native-track-player: 1.1.6

# Issue
Platform: iOS 12 (Haven't tested it on android)
When seekTo is called it doesn't update Trackplayer's position.

# Code
Suppose current position is 20 and if I do the below
```
await TrackPlayer.seekTo(1000)
const newPosition = await TrackPlayer.getPosition()
console.log(newPosition)

// Prints 20 and after a moment it updates to 1000.
```

Expected behavior
---
The TrackPlayer.getPosition should return updated value if I call await seekTo(1000) or else there should be an event `is-seeking`.

Other
---
As a workaround I have kept a condition in didUpdate which does the check and handles the synchronization part, but if an event `is-seeking` is provided it would be much more helpful.

**Comment by mechanizzed** [comment]
Same problem here.

I need to update my position for pass to my progress bar update the value.

**Comment by mechanizzed** [comment]
UPDATE
`"react-native-track-player": "^1.2.0",`


In this version **1.2.0** I had no more problems, after calling seekTo, I got the current updated position.

**Comment by nishanBende** [comment]
@mechanizzed It works for local audio files but for remote files it's still causing the same issue for me. 
Did you try with remote audio?

**Comment by mechanizzed** [comment]
@nishanBende Yes, all the files is from remote api.

**Comment by nishanBende** [comment]
@mechanizzed yes, I tried creating a separate project with 1.1.8 and it worked perfectly. However, I have noticed strange behaviour.

` let track = {
    url: 'http://traffic.libsyn.com/joeroganexp/p1355.mp3',
    id: '1',
    title: 'abc',
    artist: 'abc',
  };
  await TrackPlayer.add(track);
  await TrackPlayer.play();
  const oldPosition = await TrackPlayer.getPosition();

  TrackPlayer.addEventListener('playback-state', async data => {
    console.log('state ', data);

    if (data.state === 'ready') {
      await TrackPlayer.seekTo(5030);
      await TrackPlayer.seekTo(6000);
      const newPosition = await TrackPlayer.getPosition();
      console.log('positions ', oldPosition, newPosition);
    }
  });`

`Console logs`

`
data  {state: "loading"}
data  {state: "ready"}
data  {state: "playing"}
position  0 6000
data  {state: "paused"}
`
This returned the new position, but it paused the player, i am not sure why, i have seen it happening frequently that if you seek, it pauses the player. 
Have you come across something like that?
